contents/handbook/future.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Future
+sidebar: Handbook
+showTitle: true
+---
+
+**TL;DR:** $100 million ARR by 2026, working backwards
+## Will PostHog sell?
+
+What motivates us is building an epic product and company.
+
+We're excited by:
+
+* figuring out how far we can go
+* helping engineers build products - reduce the need for product and data teams
+* beating all the point solution competitors
+* having customers buy _from_ us instead of us selling _to_ them
+
+We're *not* excited by:
+
+* building a $1 billion company - we think going _much_ further is where the real challenge lies
```

Comment:
Very subtle but the phrasing of this could mean you want to build a company that is smaller than $1bn
```suggestion
* selling the company for $1 billion - we think we can build a much bigger company by staying independent
```